OpenRA.Mods.Common/Scripting/Properties/CaptureProperties.cs

```diff
@@ -0,0 +1,43 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptPropertyGroup("Targeting")]
+	public class CaptureProperties : ScriptActorProperties, Requires<CapturesInfo>
+	{
+		public CaptureProperties(ScriptContext context, Actor self)
+			: base(context, self) { }
+
+		[Desc("Captures the target actor using normal capturing logic.")]
+		public void Capture(Actor target)
+		{
+			Self.World.IssueOrder(new Order("CaptureActor", Self, true) { TargetActor = target });
+		}
+	}
+
+	[ScriptPropertyGroup("Targeting")]
+	public class ExternalCaptureProperties : ScriptActorProperties, Requires<ExternalCapturesInfo>
+	{
+		public ExternalCaptureProperties(ScriptContext context, Actor self)
+			: base(context, self) { }
+
+		[Desc("Captures the target actor using external-capturing logic.")]
+		public void ExternalCapture(Actor target)
```

Comment:
It would be nicer if this was also named `Capture`, the same way that `Mobile` and `Aircraft` expose alternative `Move` script properties, and the different versions of `Attack`.